rust/operator/src/hdfs_controller.rs

```diff
@@ -302,23 +300,25 @@ fn rolegroup_statefulset(
     let init_containers;
     let containers;
 
+    let (pvc, rr) = hdfs.resources(role, rolegroup_ref);
```

Comment:
Can we change the variable `rr` to `resources` or `resource_requirements`?

Reply:
Done.